### PR TITLE
fix: remove extra spaces

### LINKED ISSUE #140: Extra spaces in description
In some cases, description contains tabulations or extra spaces. It should be removed from description.

Also, when description multiline, lines concate without spaces. Please, add it.

### PR DESCRIPTION
**Pull request by ifedchankau**
- Remove extra spaces from description of argument
- Fix parsing of multiline descriptions

Closes #140

### CODE REVIEW

File: src/sections.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const splitOptionsSection = (section) => {
         if (string.indexOf('-') === 0) {
             result.push(splitStringWithArguments(string));
         } else if (result.length > 0) {
-            result[result.length - 1].description += string;
+            result[result.length - 1].description += (' ' + string);
```
Comment (romanrostislavovich) [inline src/sections.js]: I suggest to add space an end of line and remove brackets

Comment (ifedchankau) [reply src/sections.js]: @svoboda-rabstvo done.

**Review by romanrostislavovich — APPROVED** [review]
+1

**Pull request merged**